sources/packages/web/src/views/institution/designations/DesignationRequest.vue

```diff
@@ -79,6 +68,27 @@ export default {
             requestForDesignation: false,
           } as DesignationLocationsListItem),
       );
+
+      designationModel.value = {
+        institutionName: institutionState.value.legalOperatingName,
+        institutionType: institutionState.value.institutionType,
+        isBCPrivate: institutionState.value.isBCPrivate,
+        viewMode: DesignationFormViewModes.submission,
+        locations: designationModelLocations,
+        dynamicData: {
```

Comment:
Dynamic data is assigned inside and outside the if. why so ?

Reply:
nice catch. left over from my test